packages/dd-trace/test/format.spec.js

```diff
@@ -257,22 +258,12 @@ describe('format', () => {
       expect(trace.meta[ORIGIN_KEY]).to.equal('synthetics')
     })
 
-    it('should add runtime tags', () => {
-      spanContext._tags['service.name'] = 'test'
-
+    it('should add the language tag for a basic span', () => {
```

Comment:
There should be an additional test to validate that it's added only when needed.

Reply:
Now adding the language tag on all spans so I don't believe another test case is necessary.

Reply:
Please check your slack direct messages for more information on language tag behavior.